common/options/options.go

```diff
@@ -495,7 +516,7 @@ func LogSensitiveOptionWarnings(args []string) {
 
 	// Create temporary options for parsing command line args.
 	tempOpts := New("", "", "", "", true, EnabledOptions{Auth: true, Connection: true, URI: true})
-	extraArgs, err := tempOpts.parser.ParseArgs(args)
+	extraArgs, err := tempOpts.CallArgParser(args)
```

Comment:
We technically don't need to use `CallArgParser` here since we call it on the `tempOpts` that we create here, but I'm okay to leave it as-is since it doesn't matter much in the end.

Reply:
Yea, I agree we don't need it here. But just to be consistent and emphasize we shouldn't be calling `parser.ParseArgs` directly even it's not a bug somewhere.